gradio/exceptions.py

```diff
@@ -1,3 +1,7 @@
+class DuplicateBlockError(KeyError):
```

Comment:
I think KeyError is usually reserved for when a key in a dictionary is not present? Minor point but I think a `ValueError` or `LookupError` would be more appropriate here.

Reply:
Sure, will change to `ValueError`